### PR TITLE
✨ Add Hangman Game

### PR DESCRIPTION
**Pull request by Koch-Dustin**
Changes:
   - Add Hangman

PR: #1

### CODE REVIEW

File: Hangman.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      print("übrige versuche: " + str(tries))
+      break
+    
+  if tries == 0:
+    msg = "Das gesuchte Wort war: " + word
+    print(msg)
+    print("Leider verloren, vielleicht ja nächstes mal")
+    
+setUpVariables()
+play(GetRandomWordFromWordList())
```
Comment (Nicojele) [inline Hangman.py:86]: add End of File

File: HangmanStages.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            ,
+            "        --------\n" 
+            "        |       \n" 
+            "        |       \n"
+            "        |        \n"
+            "        |       \n"
+            "        |         \n"
+            "        -         "
+            ]
+      return stages[tries]
```
Comment (Nicojele) [inline HangmanStages.py:75]: add End of File

**Review by Nicojele — CHANGES_REQUESTED** [review]

File: Hangman.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  global guessed
+  global tries
+  global secret_word
+  count = 0
+  already_guessed = []
+  guessed = False
+  tries = 8
+  
+
+
```
Comment (DevBastiS) [inline Hangman.py:31]:
```suggestion
  
```

File: Hangman.py [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+      if guess.isalpha():
+        clear()
+        tries -= 1
+        print(getHangmanStages(tries))
+        msg = "Das gesuchte Wort enthält den Buchstaben '" + guess + "' nicht"
+      else:
+        msg = "Es sind nur Buchstaben erlaubt!"
+        
+    
+
```
Comment (DevBastiS) [inline Hangman.py:71]:
```suggestion
        
```

**Review by DevBastiS — APPROVED** [review]

**Review by Nicojele — APPROVED** [review]

**Pull request merged**